pyproject.toml

```diff
@@ -1,12 +1,10 @@
 [project]
 name = "modalities"
 version = "0.3.2"
```

Comment:
Why did we remove this? Our testing is always against 3.10 and 3.11. Do we need a more recent python version?